auth/strategies.local.js

```diff
@@ -82,4 +82,24 @@ module.exports = function(passport) {
 		});
 
    }));
+
+	passport.use('change-password', new LocalStrategy({
```

Comment:
Not sure how this is different from the passport.use('local-login') path. To validate if the password the person gave is correct, can't we just use the local-login method instead?